README.md

```diff
@@ -89,7 +89,21 @@ qmgrConnectionParams:
 # How long to wait until metrics are published by queue manager (milliseconds).
 # Value must be at least 10000 (periodicity with which metrics are published by MQ).
   connTimeout: 12000
-
+# Use TLS connection to queue manager?
+  useTLS: true
+# Path to keystore file
+  keystorePath: /opt/mq_exporter/keystores/keystore.jks
+# keystore password
```

Comment:
Could you please start comments in config file with a capital letter (to comply with the general style).